mava/systems/tf/mappo/system.py

```diff
@@ -190,12 +190,15 @@ def __init__(
         self._network_factory = network_factory
         self._logger_factory = logger_factory
         self._environment_spec = environment_spec
-        # Setup agent networks
+        # Setup agent networks assign a single network to all agents of the same type
```

Comment:
This doesn't read too nicely. Perhaps a line break between "networks" and "assign", or think about rephrasing.

Reply:
Updated the comments to be more explicit

Reply:
Thanks 🙂